src/id-card/index.html

```diff
@@ -223,6 +223,7 @@
                     },
                     saveIdCard: function () {
                         console.log('saveIdCard');
+                        this.idCardToIssue['revocationKey'] = `${luxon.DateTime.now().ts}`;
```

Comment:
Would not be better to use as a `key` something like `version1` or `pocSS` or something that is really a key that is repeatable? I think we do a better use of the key if we identify it as some "group" or "cluster" of the given credential issued by this key.

That use above is more suited to the `issued` attribute.

Reply:
I don't think so! The key should be something unique, otherwise, you won't be able to revoke it! as current irma cli subcommand doesn't accept `issued`! So, for example, if issue all your credential with the same key, when you want to revoke one of them, you would revoke all of them!
BTW, `issued` is generated by irma server!

Reply:
We may need to do more tests about that, but I am not sure that it must always have a totally unique value as we have alternatives. Document says:

> For example, the following RevocationRequest instructs the server to revoke the irma-demo.MijnOverheid.root instance to which it previously assigned bsn-12345 as revocationKey during issuance, and which was issued at Unix nano timestamp 1583765731750425000. If issued is not specified, all previously issued credentials with matching revocationKey are revoked.

I can see that is expected to have `revocationKey` to revoke a set of credentials with the same key and a more particular credential using the key and issued properties. If we use the key as a group, we can, for instance, make demos showing that we can revoke the IDNZ card from 2021 if we create a key name `v2021`. And, at the same time, we can show that we can revoke a specific IDNZ card from a member that decided to cancel his card using the key and the issued property that we have stored on database.

Reply:
But we would rely on irma-front-end, right? We have an endpoint that accepts both `revocationKey` and `issued`.

Reply:
That endpoint doesn't work either! I've already mentioned that here:https://internetnz.atlassian.net/wiki/spaces/RCENG/pages/1011581103/IRMA+-+Feature+-+Revocation#revoking-credential
The error is known to IRMA and they are working on that!

Reply:
I believe that's been the plan since we knew what parts of it are working and what parts are not! But I would say the overall feature is not production-ready yet.

Reply:
Yeah, I see you. No problem. As the title says `add revocation` and we did not necessarily mention the limitations on the implementation, I wanted to be sure about what exactly we are implementing (and what we are not).

Symptoms of working with something at the edge. :-D